src/w3iProxy/chatProviders/internalChatProvider.ts

```diff
@@ -15,6 +16,17 @@ export default class InternalChatProvider implements W3iChatProvider {
   public constructor(emitter: EventEmitter) {
     this.emitter = emitter
 
+    interval(2000).subscribe(() => {
+      if (!this.chatClient) {
+        return
+      }
+
+      if (!this.chatClient.core.relayer.connected) {
+        // Ping empty topic to trigger reconnection mechanism
```

Comment:
Assuming this is a workaround until https://github.com/WalletConnect/walletconnect-monorepo/pull/2108 has been released?